R/surv-roc_survival_curve.R

```diff
@@ -132,6 +154,8 @@ roc_curve_survival_impl <- function(truth,
   )
   specificity <- cumsum(specificity)
   specificity <- specificity / specificity_denom
+  specificity <- dplyr::if_else(specificity > 1, 1, specificity)
```

Comment:
Does this happen?

Reply:
the changes in this PR made it so the last element of `specificity` for `eval_time = 200` was `2*10^-12` more than 1...